code/game/objects/items/weapons/implants/implant_mindshield.dm

```diff
@@ -19,11 +19,6 @@
 
 /obj/item/implant/mindshield/implant(mob/target)
 	if(..())
-		if(target.mind in SSticker.mode.head_revolutionaries || is_shadow_or_thrall(target))
-			target.visible_message("<span class='warning'>[target] seems to resist the implant!</span>", "<span class='warning'>You feel the corporate tendrils of Nanotrasen try to invade your mind!</span>")
-			removed(target, 1)
-			qdel(src)
-			return -1
```

Comment:
Removing the logic for the head rev too? I know revs aren't regularly played, but still, seems unrelated.